extensions/markdown-language-features/package.json

```diff
@@ -353,6 +353,7 @@
   },
   "dependencies": {
     "highlight.js": "^10.4.1",
+    "insane": "^2.6.2",
```

Comment:
In this past I've used [dompurify](https://github.com/cure53/DOMPurify) which seems better maintained, more widely used, and has a bug bounty program. It seems like there may be [a vulnerability with insane](https://github.com/bevacqua/insane/issues/19).